Update design-for-evolution.md (#4524)
MicrosoftDocs/architecture-center: docs/guide/design-principles/design-for-evolution.md

**Design and test against service contracts**. When services expose well-defined APIs, you can develop and test against those APIs. That way, you can develop and test an individual service without spinning up all of its dependent services. (Of course, you would still perform integration and load testing against the real services.)

> **Use fitness functions**. Fitness functions measure the outcome to see whether it's closer or further away from an optimal solution. Fitness functions help to protect architectural characteristics as change occurs over time. Fitness function is any mechanism that provides an objective integrity assessment of architecture characteristics. The assessment may include a variety of mechanisms such as metrics, unit tests, chaos engineering etc depending on the context. For example, the architect may identify page load time as an important characteristic. Subsequently, the architect should build a fitness function to test page load time and run the test as part of continuous integration. 

Reviewer: ```suggestion
**Use fitness functions**. Fitness functions measure the outcome to see whether it's closer or further away from an optimal solution. Fitness functions help to protect architectural characteristics as change occurs over time. Fitness function is any mechanism that provides an objective integrity assessment of architecture characteristics. The assessment may include a variety of mechanisms such as metrics, unit tests, chaos engineering, and so on. For example, the architect may identify page load time as an important characteristic. Subsequently, the workload should have a fitness function to test page load time and run the test as part of continuous integration.
```
